### ISSUE TITLE
基于merge.json训练了3轮，效果较checkpoint-final差很多，没有改动参数，求指导。

### ISSUE THREAD

**Issue by xienan0326** [Facico/Chinese-Vicuna#92]

**Comment by Facico** [comment]
硬件配置是什么？loss都正常吗？

**Comment by xienan0326** [comment]
> 硬件配置是什么？loss都正常吗？
40g的A100，loss正常，如下所示。 
{'loss': 0.9024, 'learning_rate': 3.563218390804597e-05, 'epoch': 0.89}
{'eval_loss': 0.8243093490600586, 'eval_runtime': 0.3851, 'eval_samples_per_second': 2.597, 'eval_steps_per_second': 2.597, 'epoch': 0.89}         
{'loss': 0.9073, 'learning_rate': 3.333333333333333e-05, 'epoch': 0.89}
{'loss': 0.9076, 'learning_rate': 3.1034482758620685e-05, 'epoch': 0.9}
{'loss': 0.9148, 'learning_rate': 2.8735632183908045e-05, 'epoch': 0.91}
{'loss': 0.9093, 'learning_rate': 2.6436781609195398e-05, 'epoch': 0.91}
{'loss': 0.9006, 'learning_rate': 2.4137931034482755e-05, 'epoch': 0.92}
{'loss': 0.9077, 'learning_rate': 2.1839080459770115e-05, 'epoch': 0.93}
{'loss': 0.9055, 'learning_rate': 1.9540229885057468e-05, 'epoch': 0.94}
{'loss': 0.9076, 'learning_rate': 1.7241379310344825e-05, 'epoch': 0.94}
{'loss': 0.9092, 'learning_rate': 1.4942528735632182e-05, 'epoch': 0.95}
{'loss': 0.9058, 'learning_rate': 1.264367816091954e-05, 'epoch': 0.96}
{'eval_loss': 0.8262965679168701, 'eval_runtime': 0.3839, 'eval_samples_per_second': 2.605, 'eval_steps_per_second': 2.605, 'epoch': 0.96}         
{'loss': 0.9008, 'learning_rate': 1.0344827586206895e-05, 'epoch': 0.97}
{'loss': 0.9061, 'learning_rate': 8.045977011494252e-06, 'epoch': 0.97}
{'loss': 0.9065, 'learning_rate': 5.747126436781608e-06, 'epoch': 0.98}
{'loss': 0.9006, 'learning_rate': 3.448275862068965e-06, 'epoch': 0.99}
{'loss': 0.9073, 'learning_rate': 1.1494252873563217e-06, 'epoch': 1.0}

**Comment by xienan0326** [comment]
> 硬件配置是什么？loss都正常吗？
finetune.sh
![image](https://user-images.githubusercontent.com/38122102/233091238-08fd849e-6e58-4939-9e76-005c93db8fbc.png)

**Comment by Facico** [comment]
参数跟我们一样，3个epoch的step应该是checkpoint-17200左右，可以看看你的数据merge.json有没有问题?或者batch size是128吗

**Comment by xienan0326** [comment]
> 参数跟我们一样，3个epoch的step应该是checkpoint-17200左右，可以看看你的数据merge.json有没有问题?或者batch size是128吗

您好，请教一下，all_data指的是哪些数据集，因为final版本测试起来，挺不错的，想复现学习一下，plz。
![image](https://user-images.githubusercontent.com/38122102/233239221-34dcca89-9bc8-4e14-bc16-838d8a2e3257.png)

**Comment by xienan0326** [comment]
> 参数跟我们一样，3个epoch的step应该是checkpoint-17200左右，可以看看你的数据merge.json有没有问题?或者batch size是128吗

您好，请教一下，all_data指的是哪些数据集，因为final版本测试起来，挺不错的，想复现学习一下，plz。
![image](https://user-images.githubusercontent.com/38122102/233239221-34dcca89-9bc8-4e14-bc16-838d8a2e3257.png)

**Comment by Facico** [comment]
这个all_data是belle 0.5M和guanaco的数据。其实你的问题看起来像是训练的还不太够？我们checkpoint-final是17200step的，你上面测的是2600step的

**Comment by xienan0326** [comment]
> 参数跟我们一样，3个epoch的step应该是checkpoint-17200左右，可以看看你的数据merge.json有没有问题?或者batch size是128吗
这是finetune.py的参数，有一个问题，3个EPOCHS，您花费了多长时间？
![image](https://user-images.githubusercontent.com/38122102/233239921-03af0605-f605-4bc0-9723-917a185a2c23.png)
![image](https://user-images.githubusercontent.com/38122102/233239955-fec12af3-5259-4672-a83e-ed6aa575542b.png)

**Comment by grantchenhuarong** [comment]
基于LLaMA-7B32层网络的中文小羊驼，制作了43万指令问答语料，在checkpoint-11600的Lora模型基础上，使用2080Ti11GB显卡训练。显存占用10GB，75秒左右一轮，需要执行17298-11600=5698轮，预期5天左右完成训练。
